packages/clarity-js/src/layout/schema.ts

```diff
@@ -51,6 +53,12 @@ export function ld(json: any): void {
     }
 }
 
-function num(input: string): number {
-    return input ? Math.round(parseFloat(input)) : null;
+function num(input: string | number, scale: number = 1): number {
+    if (input !== null) {
+        switch (typeof input) {
+            case Constant.Number: return Math.round((input as number) * scale);
+            case Constant.String: return Math.round(parseFloat((input as string).replace(digitsRegex, Constant.Empty)) * scale);
```

Comment:
Shouldn't this be done in another higher layer if we expect this it have non-numeric parts?